selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Preferences.java

```diff
@@ -370,6 +414,45 @@ public void typeAndWaitEmailCommitter(String emailCommitter) {
     waitInputEmailCommitter(emailCommitter);
   }
 
+  /** click on the 'Contribute' checkbox */
+  public void clickOnContributeCheckbox() {
+    webDriverHelper.waitAndClick(setContributeCheckbox);
+  }
+
+  /** call the 'Contribute' by hot key */
+  public void callContributeActionByHotKey() {
+    webDriverHelper.sendKeys(Keys.chord(CONTROL, ALT, "6"));
+  }
+
+  /** wait the 'Contribute' checkbox is selected */
+  public void waitContributeCheckboxIsSelected() {
+    webDriverHelper.waitElementIsSelected(showContributeCheckbox);
+  }
+
+  /** wait the 'Contribute' checkbox is not selected */
+  public void waitContributeCheckboxIsNotSelected() {
+    webDriverHelper.waitElementIsNotSelected(showContributeCheckbox);
+  }
+
+  /**
+   * Set 'Contribute' checkbox to specified state and wait it state
+   *
+   * @param state state of checkbox (true if checkbox must be selected)
+   */
+  public void setAndWaitStateContributeCheckbox(boolean state) {
```

Comment:
`wait` looks redundant in method name
How about simplified name `setContributeCheckbox(boolean checked)`